migrations/app/20171021202147_app_workload_points_calcuations_gs_field_non_null.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function (knex, Promise) {
+  var sql = 'UPDATE app.workload_points_calculations SET gs_adjustment_points = 0 WHERE gs_adjustment_points IS NULL;'
```

Comment:
These fields had a default value of 0 set, so should there have been nulls in the table?

Reply:
There shouldn't be but they are because these fields didn't exist in the old system. Also in order to index the view it must be a non-nullable constraint.